pkgs/applications/virtualization/vmware-workstation/default.nix

```diff
@@ -0,0 +1,419 @@
+{ stdenv, buildFHSUserEnv, fetchurl, lib, zlib, gdbm, bzip2,
+libxslt, libxml2, libuuid, readline, xz, cups, glibc, libaio,
+vulkan-loader, alsa-lib, libpulseaudio, libGL, numactl, libX11,
+libXi, kmod, python3, autoPatchelfHook, makeWrapper, sqlite,
+enableInstaller ? false }:
+
+with lib;
+
```

Comment:
```suggestion
```
Using with over the entire file is an anti pattern.

Reply:
`with lib;` removed